### PR TITLE
added aci_static_binding_to_epg module

### PR DESCRIPTION
**Pull request by brunocalogero**
##### SUMMARY
added aci_static_binding_to_epg module (fv:RsPathAtt)

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
aci_static_binding_to_epg

##### ANSIBLE VERSION
```
2.5
```

##### ADDITIONAL INFORMATION
Also resolves the following issue: https://github.com/ansible/ansible/issues/35340

### CODE REVIEW

**Comment by brunocalogero** [comment]
@dagwieers man.. these documentation errors :P is it because I have to provide either "choices" or "aliases" for every parameter in the doc ?

**Comment by dagwieers** [comment]
@brunocalogero Well, it means you have a YAML error in the documentation. What I usually do is copy&paste into yamllint to find the cause.

File: lib/ansible/modules/network/aci/aci_static_binding_to_epg.py [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+    default: regular
+  connection_type:
+    description:
+    - The Path Type for the static EPG deployement.
+    - The APIC defaults the Path Type to C(access_interface) which is the same as C(Port).
+    choices: [ access_interface, virtual_port_channel, direct_port_channel, fex ]
+    default: access_interface
+  pod:
+    description:
+    - the pod number part of the tDn.
```
Comment (dagwieers) [inline lib/ansible/modules/network/aci/aci_static_binding_to_epg.py]: Start with a capital.

File: lib/ansible/modules/network/aci/aci_static_binding_to_epg.py [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+    - C(pod) is usually an integer below 10
```
Comment (dagwieers) [inline lib/ansible/modules/network/aci/aci_static_binding_to_epg.py]: End with a dot.

File: lib/ansible/modules/network/aci/aci_static_binding_to_epg.py (hunk 1 above)
Comment (dagwieers) [reply lib/ansible/modules/network/aci/aci_static_binding_to_epg.py]: Do the same for other descriptions.

**Review by dagwieers — CHANGES_REQUESTED** [review]

**Pull request merged**